qCC/db_tree/ccPropertiesTreeDelegate.cpp

```diff
@@ -2341,6 +2368,7 @@ void ccPropertiesTreeDelegate::octreeDisplayedLevelChanged(int val)
 		octree->setDisplayedLevel(val);
 		updateDisplay();
 		//we must also reset the properties display!
+		m_lastFocusItemRole = OBJECT_OCTREE_LEVEL;
```

Comment:
I guess these lines shouldn't be below the comment which applies to the call to 'updateModel()'. Can you move them after updateModel() or before the comment?

Reply:
> 		//record item role here to use it 'setFocus' in 'createEditor', 
> 		//than you can continuous scroll the widget's value.
> 		m_lastFocusItemRole = OBJECT_CONE_HEIGHT;
> 
> 		//we must also reset the properties display!
> 		updateModel();
is this ok? Please let me know if there is a problem with the expression .

Reply:
Thanks for asking. I would simply write something like:
//record item role to force the scroll focus (see 'createEditor').